test/unit/mock-filename-generator-spec.js

```diff
@@ -63,7 +63,26 @@ describe('mock filename generator', function() {
 
     var mockResponsePath = mockFilenameGenerator.getMockPath(prism, req);
 
-    assertPathEqual(mockResponsePath, 'mocks/foo/_is_this_url_really=that_237a2df1a4dfefe26d84b37103583200b8cd9c48.json');
+    assertPathEqual(mockResponsePath, 'mocks/foo/_is_this_url_237a2df1a4dfefe26d84b37103583200b8cd9c48.json');
```

Comment:
My bad, that's the resultant URL and the behavior has changed. Thanks for the help!